server/package.json

```diff
@@ -13,10 +13,12 @@
     "connect-flash": "^0.1.1",
     "connect-session-sequelize": "^3.0.0",
     "cookie-parser": "^1.4.1",
+    "ejs": "^2.4.1",
     "express": "^4.12.4",
     "express-session": "^1.13.0",
     "istanbul": "^0.4.2",
     "jasmine": "^2.4.1",
+    "mysql": "^2.10.2",
```

Comment:
@NoSkiiilL @phong-steph 
`mysql` should not be included as a project dependency, but as a project development dependency.
Tip: Use `npm install --save-dev myPackage` instead of `npm install --save myPackage`.